src/openforms/submissions/templates/admin/submissions/submission/logs.html

```diff
@@ -17,21 +17,34 @@
 {% endblock %}
 
 {% block content %}
-    <h1>{% trans  "Logic logs" %}</h1>
     {% block field_sets %}
         {% if not logs_activity %}
             {% trans "No logic rules for that submission" %}
         
         {% else %}
             {% for log in logs_activity  %}
-                <span class="timestamp-log"></span>
                 <fieldset class="module aligned collapse collapsed fieldset-log">
                     <h2>
                         <time datetime="{{ log.timestamp.isoformat }}" title="{{ log.timestamp|date:'Y-m-d H:i:s' }}">
                             {% blocktrans with since=log.timestamp|timesince %}{{ since }} ago{% endblocktrans %}
                         </time>
                         - {% trans 'Logic' %} -
                     </h2>
+                    <h4> {% trans "Variables" %} </h4>
+                    <table class="logic-rules-tables">
+                        <thead>
+                            <th class="property-name-width"> {% trans "Property name" %} </th>
+                            <th> {% trans "Value" %} </th>
+                        </thead>
+                        <tbody>
+                            {% for key, value in log.extra_data.involved_components.items  %}
+                            <tr>
+                                <td>{{ value.stepName }}: {{ key }} &#40;{{ value.label }}&#41;</td>
```

Comment:
```suggestion
                                <td>{{ value.stepName }}: <code>{{ key }}</code> &#40;{{ value.label }}&#41;</td>
```
+ some nice styling will make this stand out visually